tests/elevation_service/one_point_per_user.py

```diff
@@ -1,131 +1,164 @@
-        self.test_results.append({"users": self.users_count, "avg_response_time": average_response_time})
-        # print(self.test_results)
-        create_custom_graph(graph_name="Users_vs_AvgResponseTime", graph_path=reports_path,
-                            test_results=self.test_results, graph_title=None, rotation=None)
-
-        # self.req_start_t_rsp_t = create_graph_results_data_format(["start_time", "response_time"],
-        #                                                           [])
-        # print(self.req_start_t_rsp_t)
-        create_start_time_response_time_graph(graph_name=f"RequestStartTime_vs_ResponseTime-{run_number}",
-                                              start_time_data=start_time_data, response_time_data=response_time_data)
-
-        # percent_value_by_range = {}
-        # if total_requests != 0:
-        #     for index, (key, value) in enumerate(counters.items()):
-        #         range_percent_val = (value / total_requests) * 100
-        #         percent_value_by_range[f"{percent_ranges[index]}"] = range_percent_val
-        #     percent_value_by_range["total_requests"] = total_requests
-        #     write_rps_percent_results(
-        #         custom_path=reports_path, percente_value_by_range=percent_value_by_range
-        #     )
-        # else:
-        #     percent_value_by_range["total_requests"] = total_requests
+        if not retype_env(ElevationConfig.token_flag):
+            self.client.post(
+                "/",
+                json=body,
+                headers={"Content-Type": "application/json"},
+                verify=False,
+            )
+        else:
+            self.client.post(
+                f"?token={ElevationConfig.TOKEN}",
+                json=body,
+                headers={"Content-Type": "application/json"},
+            )
 
+    # def on_stop(self):
+    #     # print(self.body)
+    #     average_response_time = self.environment.runner.stats.total.avg_response_time
+    #     # create_start_time_response_time_graph(graph_name=f"RequestStartTime_vs_ResponseTime-{run_number}",
+    #     #                                       start_time_data=start_time_data, response_time_data=response_time_data)
+    #
+    #     # print(self.request_bodies_cycle)
 
-counters = initiate_counters_by_ranges(config_ranges=percent_ranges)
-total_requests = 0
-run_number = 0
-start_time_data = []
-response_time_data = []
-file_created = False
 
+@events.test_start.add_listener
+def on_locust_init(environment, **_kwargs):
+    environment.users_count = environment.runner.target_user_count
 
-@events.request.add_listener
-def on_request(response_time, **kwargs):
-    start_time = datetime.datetime.fromtimestamp(kwargs['start_time'])
-    response_time_data.append(response_time)
-    start_time_data.append(start_time)
+
+counters = initiate_counters_by_ranges(config_ranges=percent_ranges)
 
 
 @events.request.add_listener
 def response_time_listener(response_time, **kwargs):
```

Comment:
General comment : *kwargs is not good practice, better us meaningful names for function param :)